common/client/multi_node.go

```diff
@@ -114,6 +95,7 @@ type multiNode[
 	leaseDuration       time.Duration
 	leaseTicker         *time.Ticker
 	chainFamily         string
+	reportInterval      time.Duration
```

Comment:
I don't see any benefits in storing the const variable here. I slightly prefer the old way. Also, we can avoid having to set it on the `multi_node_test.go` (https://github.com/smartcontractkit/chainlink/pull/11066/files#diff-a9fee197ea880bb302ca4c0f830d292355586dca02d46c826becda641404cf30R232), since it's not used anywhere.